src/main/resources/application.conf

```diff
@@ -378,6 +378,9 @@ metronome {
   # (Optional. Default: 10) Maximum number of JobRun history entries.
   history.count = ${?METRONOME_HISTORY_COUNT}
 
+  # (Optional. Default: false) Enable suppressing offers
+  suppressOffers = ${?METRONOME_SUPPRESS_OFFERS}
```

Comment:
I'd put this underneath `scheduler`, i.e. make the path `metronome.scheduler.suppressOffers`.